### ISSUE TITLE
Opening terminal in focus mode and opening a second window then closing the first window with ctrl+shift+w freezes terminal

### ISSUE THREAD

**Issue by AndysonDK** [microsoft/terminal#9306]
# Environment

```none
Windows build number: Microsoft Windows [Version 10.0.19042.844]
Windows Terminal version: 1.5.10411.0
```

# Steps to reproduce

1. Open up a new fresh terminal instance
2. Toggle focus mode (not full-screen mode) and keep it in this mode during all of the next steps
3. Open up a new tab by using `ctrl+shift+t` (you now have a total of two tabs open)
4. Switch to the previous tab (the one you started out with when you opened up the terminal)
5. Close the tab by using the default keybinding to close a pane which is `ctrl+shift+w` (see this for reference https://docs.microsoft.com/en-us/windows/terminal/customize-settings/actions#close-pane)

# Expected behavior

The previous tab (the first tab) closes and focus goes to the recently second opened tab.

# Actual behavior

The whole terminal freezes and I have to either close the terminal by using `alt+F4` or terminate the process in task manager.

# Notes

This issue seems closely related to this one https://github.com/microsoft/terminal/issues/5799 but it still doesn't work for me.

**Comment by zadjii-msft** [comment]
Well. That's certainly weird. You're not using anything like focus mode, you running the Terminal full-screen? Could you share your settings? Are you using any sort of accessibility software (Narrator, NVDA)?

I'm having a hard time reproing this, but I also only have 1.6+ builds installed 😕

**Comment by AndysonDK** [comment]
Ah, you right! This issue only applies to when the terminal is in focus mode (not full-screen nor default mode).
I've updated the issue with the new information.

Here is my settings:
```Json
{
    "$schema": "https://aka.ms/terminal-profiles-schema",
    "defaultProfile": "{2c4de342-38b7-51cf-b940-2309a097f518}",
    "copyOnSelect": false,
    "copyFormatting": false,
    "launchMode": "focus",
    "profiles":
    {
        "defaults":
        {
            "useAcrylic": true,
            "acrylicOpacity": 0.8,
            "fontSize": 9,
            "cursorShape": "filledBox"
        },
        "list":
        [
            {
                "guid": "{2c4de342-38b7-51cf-b940-2309a097f518}",
                "hidden": false,
                "name": "Ubuntu",
                "source": "Windows.Terminal.Wsl",
                "startingDirectory" : "//wsl$/Ubuntu/home/anders"
            },
            {
                "guid": "{0caa0dad-35be-5f56-a8ff-afceeeaa6101}",
                "name": "Command Prompt",
                "commandline": "cmd.exe",
                "hidden": false
            },
            {
                "guid": "{61c54bbd-c2c6-5271-96e7-009a87ff44bf}",
                "name": "Windows PowerShell",
                "commandline": "powershell.exe",
                "hidden": false
            },
            {
                "guid": "{b453ae62-4e3d-5e58-b989-0a998ec441b8}",
                "hidden": false,
                "name": "Azure Cloud Shell",
                "source": "Windows.Terminal.Azure"
            }
        ]
    },
    "schemes": [],
    "actions":
    [
        { "command": "find", "keys": "ctrl+shift+f" },
        { "command": { "action": "splitPane", "split": "auto", "splitMode": "duplicate" }, "keys": "alt+shift+d" },
        {  "command": "toggleFocusMode", "keys": "shift+f11" }
    ]
}
```

**Comment by Don-Vito** [comment]
@zadjii-msft - wasn't it solved in 1.6 by https://github.com/microsoft/terminal/pull/8549?

**Comment by AndysonDK** [comment]
I've just tested it in 1.6 and it works! Should've updated!
